src/app/test-record/test-record.component.ts

```diff
@@ -0,0 +1,64 @@
+import {Component, OnInit, ChangeDetectionStrategy} from '@angular/core';
+import {select, Store} from '@ngrx/store';
+import {IAppState} from '@app/technical-record/adr-details/adr-details-form/store/adrDetailsForm.state';
+import {selectSelectedVehicleTestResultModel} from '@app/store/selectors/VehicleTestResultModel.selectors';
+import {Observable} from 'rxjs';
+import {ActivatedRoute} from '@angular/router';
+import {initAll} from 'govuk-frontend';
+import {TechRecordHelpersService} from '@app/technical-record/tech-record-helpers.service';
+import {TestResultModel} from '@app/models/test-result.model';
+import {TestType} from '@app/models/test.type';
+
+@Component({
+  selector: 'vtm-test-record',
+  templateUrl: './test-record.component.html',
+  styleUrls: ['./test-record.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TestRecordComponent implements OnInit {
+
+  testResultObservable$: Observable<any>;
+  testRecord: TestResultModel;
+  testType: TestType;
+  testTypeNumber: string;
+  seatBeltApplicable: string[] = ['1', '3', '4', '7', '8', '10', '14', '15', '16', '18', '19', '21', '22', '23', '27', '28', '93'];
+  emissionDetailsApplicable: string[] = ['44', '39', '45'];
+  defectsApplicable: string[] = ['30', '31', '32', '33', '34', '35', '36', '38', '39', '44', '45', '47', '48',
+    '49', '50', '56', '57', '59', '60', '85', '86', '87', '88', '89', '90', '100', '121'];
+
+  constructor(private _store: Store<IAppState>, private route: ActivatedRoute, public techRecHelpers: TechRecordHelpersService) {
```

Comment:
Ideally A component should not be having a reference to a store. That should be the job of a container wrapper. Hence you wont to subscribe in this file. Besides you are not unsubscribing so memory leak. Check my branch on how I have introduced TechRecordContainer. Use similar approach